portal/src/PasswordStrengthMeter.module.css

```diff
@@ -1,9 +1,11 @@
-$passwordMeterInactiveColor: #e0e0e0;
-$passwordMeterLevel1Color: #e23d3d;
-$passwordMeterLevel2Color: #e67c00;
-$passwordMeterLevel3Color: #d69900;
-$passwordMeterLevel4Color: #93a03f;
-$passwordMeterLevel5Color: #04ab18;
+:root {
```

Comment:
`:root` is HTML element. Define rules on the root element in a CSS module is ambiguous. Instead, move the custom variables definition to the topmost CSS class of this component. I guess it is `.passwordStrengthMeter`.